src/components/Trends.js

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import './Trends.css';
+import {BiSearch} from 'react-icons/bi';
+import Biden from '../images/trends-biden.jpg';
+import TrendsFood from '../images/trends-food.jpg';
+
+
+const Trends = () => {
+
+    return(
+
+    <div className="trends-panel">
+        {/* <div className="searchBar" > */}
```

Comment:
Is this a placeholder for something that will be added later?

Reply:
Its a CSS className / div className Issue which will be fixed

Reply:
Ah ok, cool :)